### PR TITLE
Add `Featured Artist Track Updates: Winter 2024` news post

### PR DESCRIPTION
**Pull request by pishifat**
- [x] placeholder links

### CODE REVIEW

File: news/2024/2024-01-07-featured-artist-track-updates-winter-2024.md [hunk 1]
```diff
@@ -0,0 +1,254 @@
...
+- **cygnus - Silver Mutation**
+- **cygnus - Silver Orbit**
+- **cygnus - St. Ophelia**
+- **cygnus - Sun Flower**
+- **cygnus - Tasogare no Megami**
+- **cygnus - Twilight Ceremony**
+
+### cygnus - Silver Mutation
+
+Try out [this osu! marathon](https://osu.ppy.sh/beatmapsets/2067206) hosted by [the man behind the music](https://osu.ppy.sh/beatmapsets/2067206)!
```
Comment (Walavouchey) [inline news/2024/2024-01-07-featured-artist-track-updates-winter-2024.md:139]:
```suggestion
Try out [this osu! marathon](https://osu.ppy.sh/beatmapsets/2067206) hosted by [the man behind the music](https://osu.ppy.sh/users/5052899)!
```

File: news/2024/2024-01-07-featured-artist-track-updates-winter-2024.md [hunk 2]
```diff
@@ -0,0 +1,254 @@
...
+
+<audio controls>
+    <source src="https://assets.ppy.sh/artists/33/Songs/HyuN_-_iRELLiA.mp3" type="audio/mpeg">
+</audio>
+
+## [SAMString](https://osu.ppy.sh/beatmaps/artists/273)
+
+[![](https://assets.ppy.sh/artists/273/header.jpg)](https://osu.ppy.sh/beatmaps/artists/273)
+
+osu! has been investing in [osu! originals](/wiki/osu!_originals) for the past few years, but that doesn't only cover world cups — community tournaments have taken up the mantle of commissioning original songs too, including this one by [**SAMString**](https://osu.ppy.sh/beatmaps/artists/273)for the *osu!mania 4K Chinese National Cup 2023*!
```
Comment (Walavouchey) [inline news/2024/2024-01-07-featured-artist-track-updates-winter-2024.md:175]:
```suggestion
osu! has been investing in [osu! originals](/wiki/osu!_originals) for the past few years, but that doesn't only cover world cups — community tournaments have taken up the mantle of commissioning original songs too, including this one by [**SAMString**](https://osu.ppy.sh/beatmaps/artists/273) for the *osu!mania 4K Chinese National Cup 2023*!
```

File: news/2024/2024-01-07-featured-artist-track-updates-winter-2024.md [hunk 3]
```diff
@@ -0,0 +1,254 @@
...
+
+One of osu!'s first Featured Artists makes a return, this time with just *one* new track. You don't have much choice here, so start mapping already:
+
+### HyuN - iRELLiA
+
+<audio controls>
+    <source src="https://assets.ppy.sh/artists/33/Songs/HyuN_-_iRELLiA.mp3" type="audio/mpeg">
+</audio>
+
+## [SAMString](https://osu.ppy.sh/beatmaps/artists/273)
```
Comment (Walavouchey) [inline news/2024/2024-01-07-featured-artist-track-updates-winter-2024.md:171]: there are 4 new tracks but you've listed one

image

File: news/2024/2024-01-07-featured-artist-track-updates-winter-2024.md (hunk 2 above)
Comment (Hiviexd) [inline news/2024/2024-01-07-featured-artist-track-updates-winter-2024.md:175]:
```suggestion
osu! has been investing in [osu! originals](/wiki/osu!_originals) for the past few years, but that doesn't only cover world cups — community tournaments have taken up the mantle of commissioning original songs too, including this one by [**SAMString**](https://osu.ppy.sh/beatmaps/artists/273) for the *osu!mania 4K Chinese National Cup 2023*!
```

**Review by Hiviexd — CHANGES_REQUESTED** [review]

**Review by Walavouchey — CHANGES_REQUESTED** [review]

File: news/2024/2024-01-07-featured-artist-track-updates-winter-2024.md (hunk 3 above)
Comment (pishifat) [reply news/2024/2024-01-07-featured-artist-track-updates-winter-2024.md:171]: 3 are old, but the artist gave album art for them, so they show up as new. i can't un-newify them :(

**Review by Hiviexd — DISMISSED** [review]

File: news/2024/2024-01-07-featured-artist-track-updates-winter-2024.md [hunk 4]
```diff
@@ -0,0 +1,254 @@
...
+  - [**Umeboshi Chazuke**](#umeboshi-chazuke)
+- Ranked/Loved maps for new songs are listed here, along with one or two audio previews per artist. Check the artist's listing for all previews and beatmap template downloads!
+- Any new songs are marked with a **NEW** icon on the Featured Artist listing
+
+![](/wiki/shared/news/2022-01-02-featured-artist-track-updates-winter-2022/new.png)
+
+Let's get started.
+
+---
+
```
Comment (Walavouchey) [inline news/2024/2024-01-07-featured-artist-track-updates-winter-2024.md:34]: line above heading doesn't really look that great

image


```suggestion
```

**Review by Walavouchey — APPROVED** [review]

**Pull request merged**